client/client.go

```diff
@@ -389,7 +391,15 @@ func (c *CfClient) retrieveSegments(ctx context.Context) error {
 }
 
 func (c *CfClient) setAnalyticsServiceClient(ctx context.Context) {
+
 	<-c.authenticated
+	c.mux.RLock()
+	defer c.mux.RUnlock()
+	if !c.config.enableAnalytics {
+		log.Println(time.Now().Format("2006-01-02 15:04:05") + " Posting analytics data disabled.")
```

Comment:
I think for consistency we should use the `c.config.Logger` that gets used elsewhere in this client code for here and the line below. I think it logs out timestamp info for us as well so all you'll need to do is include the message https://github.com/harness/ff-golang-server-sdk/pull/96/files#diff-bd3a55a72186f59e2e63efb4951573b2f9e4a7cc98086e922b0859f8ccc1dd09L387